ext/GraphNeuralNetworksSimpleWeightedGraphsExt/GNNGraphs/gnngraph.jl

```diff
@@ -0,0 +1,23 @@
+__precompile__(false)
```

Comment:
```suggestion

```

Reply:
I'm afraid it will yield the precompiling warning about overwriting the method from GNN package.

Reply:
update: not only warning but
```julia
WARNING: Method definition (::Type{GraphNeuralNetworks.GNNGraphs.GNNGraph{T} where T<:Union{AbstractArray{T, 2} where T, Tuple{T, T, V} where V where T<:(AbstractArray{var"#s4", 1} where var"#s4"<:Integer)}})(T) where {T<:(Graphs.AbstractGraph{T} where T)} in module GNNGraphs at /home/agata/.julia/dev/GraphNeuralNetworks/src/GNNGraphs/gnngraph.jl:173 overwritten in module GraphNeuralNetworksSimpleWeightedGraphsExt at /home/agata/.julia/dev/GraphNeuralNetworks/ext/GraphNeuralNetworksSimpleWeightedGraphsExt/GNNGraphs/gnngraph.jl:3.
ERROR: Method overwriting is not permitted during Module precompilation. Use `__precompile__(false)` to opt-out of precompilation
```